cmd/krew/cmd/index.go

```diff
@@ -75,7 +75,14 @@ var indexAddCmd = &cobra.Command{
 		if !indexoperations.IsValidIndexName(name) {
 			return errInvalidIndexName
 		}
-		return indexoperations.AddIndex(paths, name, args[1])
+		err := indexoperations.AddIndex(paths, name, args[1])
+		if err != nil {
+			return err
+		}
+		internal.PrintWarning(os.Stderr, `You have added a new index from %q
+The plugins in this index are not audited for security by the Krew maintainers.
+Run them at your own risk.`+"\n", args[1])
```

Comment:
It might be better to say `Install them at your own risk.` for this particular msg.